docs/TESTING.md

```diff
@@ -28,13 +28,13 @@ Note: This configuration file must be in [JSON format](http://www.w3schools.com/
 Note: Unfortunately JSON format is not allowing you to have comments inside JSON code.
 
 Let’s see some example and let's try to configure small "test farm" with three devices connected to your host computer. In this example no peripherals (like SD card or EEPROM) are connected to our Mbed boards. We will use three platforms in this example:
-* [NXP LPC1768](https://mbed.org/platforms/mbed-LPC1768) board. 
```

Comment:
`git commit -p` will let you choose specific chunks if you just want to commit the non-whitespace changes.